src/wireviz.py

```diff
@@ -141,7 +142,7 @@ def create_graph(self):
                 for bla in p:
                     html = html + '<td>{}</td>'.format(bla)
                 html = html + '</tr>'
-                bgcolor = wv_colors.translate_color(x, 'hex')
+                bgcolor = ":".join(wv_colors.translate_color(x, 'hex').split(':')[:2])
                 html = html + '<tr><td colspan="{colspan}" cellpadding="0" height="6" bgcolor="{bgcolor}" border="2" sides="tb" port="{port}"></td></tr>'.format(colspan=len(p), bgcolor=bgcolor if bgcolor != '' else '#ffffff', port='w{}'.format(i))
```

Comment:
I suggest changing height="6" to height="10" for two-colored wires to match the extra thickness of the connecting curved wires. The thickness seems to be 2x the number of color stripes (black edge + base color + stripe color + base color + black edge).

Reply:
An even better improvement (to avoid the color gradient) is to build the striped wire using multiple HTML rows with the port attribute only present in the middle row for each wire. See this excerpt from the upper wire in ex07.gv where the black edges are separated rows:
<tr><td colspan="3" cellpadding="0" height="2" bgcolor="#000000" border="0" sides="tb"></td></tr>
<tr><td colspan="3" cellpadding="0" height="6" color="#ffffff" bgcolor="#00ff00" border="2" sides="tb" port="w1">
<tr><td colspan="3" cellpadding="0" height="2" bgcolor="#000000" border="0" sides="tb"></td></tr>
It might be easier to to use one row (with no border) for each color in the generic case with multiple stripes.

Reply:
SVG provides dashes ( https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray ) which provide a simple and clean way to add this.  Draw a solid wire then a dashed wire.

Reply:
I just saw your changes, if you are able to, please check out my solution and let me knnow if you have any comments. I will try to review your pull request shortly.

Reply:
I just created this PR: https://github.com/aakatz3/WireViz/pull/4 with my suggested implementation.

Reply:
@njhurst While it would be nice to do this, there is a problem: Graphviz does not support this properly. When drawing dashed lines, you cannot specify an alternate color, and you can't seem to overlay them properly. One COULD modify the SVG after the fact, but it breaks compatibility with GraphViz

Reply:
This looks very promising!

> @njhurst While it would be nice to do this, there is a problem: Graphviz does not support this properly. When drawing dashed lines, you cannot specify an alternate color, and you can't seem to overlay them properly. One COULD modify the SVG after the fact, but it breaks compatibility with GraphViz

It is indeed possible to define [multi-colored dashed lines in GraphViz](https://graphviz.org/doc/info/attrs.html#d:color) (see example [here](https://edotor.net/?engine=dot#graph%20%7B%0A%0A%09edge%20%5Bcolor%3D%22yellow%3B0.2%3Agreen%3B0.1%3Ayellow%3B0.4%3Agreen%3Ayellow%3B0.2%22%5D%0A%0A%09A%20--%20B%0A%7D%0A)); however, it might be hard to make it look good, especially making it look even between the edges and the HTML table cell (which I guess could be subdivided to simulate stripes?).